From adb0a9cc7b3fc7ae6980d07b91cc98311cade83c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 5 Dec 2024 13:51:25 +0800
Subject: [PATCH 174/274] AOSCOS: MIPS: workarounds: declare function
 prototypes

Add prototype declaration for gpio_lvds_off(), turn_off_lvds(), and
turn_on_lvds(), fixing build errors due to missing function prototypes.

Fixes: "AOSCOS: MIPS: workaround A1004/A1101/A1201/A1205 display issues"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/include/asm/mach-loongson64/workarounds.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/mips/include/asm/mach-loongson64/workarounds.h b/arch/mips/include/asm/mach-loongson64/workarounds.h
index a38cbc2a5a44..f669ab6df0eb 100644
--- a/arch/mips/include/asm/mach-loongson64/workarounds.h
+++ b/arch/mips/include/asm/mach-loongson64/workarounds.h
@@ -7,4 +7,8 @@
 #define WORKAROUND_LVDS_EC 	0x00000004
 #define WORKAROUND_LVDS_GPIO	0x00000008
 
+void gpio_lvds_off(void);
+void turn_off_lvds(void);
+void turn_on_lvds(void);
+
 #endif
-- 
2.47.1

